### DISCUSSION TITLE
Coroutine error

### DISCUSSION THREAD

**Discussion by Alex-Wenner-FHR** [h2oai/wave#981] [Q&A] [answered]
I am experiencing this coroutine error in on my processes running in the background... Not really too informative so I am unsure how to start to debug this. Any thoughts? 
![image](https://user-images.githubusercontent.com/78372056/131849355-2d2010d7-4660-416a-bacd-8c6aebe7efc3.png)

**Comment by mturoci** [comment]
Hi @AlexWennerDT, can you put up a minimal repro and file a bug if you believe it is not a user error please?

**Comment by Alex-Wenner-FHR** [comment, marked as answer]
Hi! After you all changed out the 500 for a 413 it became easier to debug and was able to get a work around in place!
